lib/matplotlib/ticker.py

```diff
@@ -2502,64 +2677,116 @@ def view_limits(self, vmin, vmax):
         return result
 
 
-class LogitLocator(Locator):
+class LogitLocator(MaxNLocator):
     """
     Determine the tick locations for logit axes
     """
 
-    def __init__(self, minor=False):
-        """Place ticks on the logit locations."""
-        self.minor = minor
+    def __init__(self, minor=False, *, nbins="auto"):
+        """
+        Place ticks on the logit locations
+
+        Parameters
+        ----------
+        nbins : int or 'auto', optional
+            Number of ticks. Only used if minor is False.
+        minor : bool, default: False
+            Indicate if this locator is for minor ticks or not.
+        """
+
+        self._minor = minor
+        MaxNLocator.__init__(self, nbins=nbins, steps=[1, 2, 5, 10])
 
-    def set_params(self, minor=None):
+    def set_params(self, minor=None, **kwargs):
         """Set parameters within this locator."""
         if minor is not None:
-            self.minor = minor
+            self._minor = minor
+        MaxNLocator.set_params(self, **kwargs)
 
-    def __call__(self):
-        """Return the locations of the ticks."""
-        vmin, vmax = self.axis.get_view_interval()
-        return self.tick_values(vmin, vmax)
+    @property
```

Comment:
Does this need to be a property?

Reply:
Yes, to keep to compatibility with the old `LogitLocator` which has a `minor` attribute.